<ui version="4.0" >
 <class>U3D_GUIClass</class>
 <widget class="QDialog" name="U3D_GUIClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>U3D_GUI</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>305</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="campar_box" >
     <property name="title" >
      <string>Camera Parameters</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" colspan="3" >
       <widget class="QGroupBox" name="campos_box" >
        <property name="title" >
         <string>Position</string>
        </property>
        <layout class="QVBoxLayout" >
         <item>
          <layout class="QHBoxLayout" >
           <item>
            <widget class="QLabel" name="posx_lab" >
             <property name="text" >
              <string>X</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="posx_lnedt" >
             <property name="text" >
              <string>0.0</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="posy_lab" >
             <property name="text" >
              <string>Y</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="posy_lnedt" >
             <property name="text" >
              <string>0.0</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="posz_lab" >
             <property name="text" >
              <string>Z</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="posz_lnedt" >
             <property name="text" >
              <string>-1.0</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0" colspan="3" >
       <widget class="QGroupBox" name="camtar_box" >
        <property name="title" >
         <string>Target</string>
        </property>
        <layout class="QVBoxLayout" >
         <item>
          <layout class="QHBoxLayout" >
           <item>
            <widget class="QLabel" name="tarx_lab" >
             <property name="text" >
              <string>X</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="tarx_lnedt" >
             <property name="text" >
              <string>0.0</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="tary_lab" >
             <property name="text" >
              <string>Y</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="tary_lnedt" >
             <property name="text" >
              <string>0.0</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="tarz_lab" >
             <property name="text" >
              <string>Z</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="tarz_lnedt" >
             <property name="text" >
              <string>0.0</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>101</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1" >
       <widget class="QGroupBox" name="lens_box" >
        <property name="title" >
         <string>Lens</string>
        </property>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="fov_lab" >
           <property name="text" >
            <string>FOV angle</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="fov_spn" >
           <property name="maximum" >
            <double>90.000000000000000</double>
           </property>
           <property name="singleStep" >
            <double>0.100000000000000</double>
           </property>
           <property name="value" >
            <double>60.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>111</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>305</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSplitter" name="splitter_4" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QDialogButtonBox" name="buttonBox" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons" >
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
